atlas/AtlasMl/atlasml/agent/__init__.py

```
@@ -0,0 +1,4 @@
+# Atlas AI Agent Module
+from .ai_agent import AIAgent
```

Ruff (0.12.2)

`.ai_agent.AIAgent` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)